concepts/structs/about.md

```diff
@@ -1,20 +1,52 @@
 # Structs
 
-Structs collect a fixed, heterogenous set of data into a single unit. Structs in Rust have these properties:
+Structs collect a fixed, heterogeneous set of data into a single unit. Structs in Rust have these properties:
 
 - Every struct type has a particular name.
 - Every member of a struct is called a **field**.
 - Every field of a struct has a particular name.
 - Visibility is tracked at the field level: each field may be independently public, private, or other[^1].
 
-# Tuples etc.
+
+# Structs Overview
+
+Structs come in three flavors: structs with named fields, tuple structs, and unit structs. 
+
+Structs are defined using the `struct` keyword, followed by the capitalized name of the type the struct is describing:
```

Comment:
```suggestion
Structs are defined using the `struct` keyword, followed by the name of the type the struct is describing:
```

Capitalization there is a convention which generates a warning if violated, but it's not intrinsically part of the language.

Reply:
```suggestion
Structs are defined using the `struct` keyword, followed by the capitalized name of the type the struct is describing. The name may then be followed by curly braces (for a standard struct), parentheses (for a tuple struct), or a semicolon (for a unit struct). 
```

Let's express the precise grammar.